app/metrics.yaml

```diff
@@ -123,6 +123,19 @@ events:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+  total_uri_count:
+    type: counter
+    description: >
+      A counter of URIs visited by the user
+    send_in_pings:
+    - metrics
```

Comment:
fly-by: please note that this is not sending data in the [`baseline`](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/baseline.md) ping, but rather in the [`metrics`](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/metrics.md) ping. I'm mentioning this since the PR title refers to the `baseline` ping. Please keep in mind that these pings, as documented, serve different purposes and have different schedules.

Any specific reason why you want to send this in the `baseline` ping anyway?

Reply:
@Dexterp37 Yeah, we noticed a couple of inconsistencies in the ticket. We're working on sorting it out!

Reply:
Ya, also @colintheshots to update the PR/issue/commit message of this to not say baseline ping.